docs/content/v0.20/packaging.md

```diff
@@ -0,0 +1,28 @@
+# Packaging and Delivery
+
+## status-addon
+
+The [status-addon](https://github.ibm.com/dettori/status-addon) repo is the source of a RedHat-style operator. 
+
+The operator is delivered by a Helm chart at `quay.io/pdettori/status-addon-chart`. Chart versioning? How is the chart built, pushed?
+
+There are probably one or more container images involved.
+
```

Comment:
There is one. Plan is to open source the status add on as well and make the image part of our official set of images.

Reply:
Updated, thanks.